source/Jobbr.Storage.MsSql/Jobbr.Storage.MsSql.csproj

```diff
@@ -20,33 +20,19 @@
   <PropertyGroup>
     <PreBuildEvent />
   </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Net.Http" />
-  </ItemGroup>
   <ItemGroup>
     <None Include="CreateTables.sql">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Jobbr.ComponentModel.JobStorage" Version="1.3.0" />
-    <PackageReference Include="Jobbr.ComponentModel.Registration" Version="1.0.1" />
-    <PackageReference Include="LibLog" Version="3.1.0">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="ServiceStack.Common" Version="5.4.0" />
-    <PackageReference Include="ServiceStack.Interfaces" Version="5.4.0" />
-    <PackageReference Include="ServiceStack.OrmLite" Version="5.4.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
     <PackageReference Include="ServiceStack.OrmLite.SqlServer" Version="5.4.0" />

```

Comment:
Can you try and see what it takes to update to 6.4.0?

Reply:
latest is 6.5.0, is there any specific reason for the version that I should know about or just typo?

Reply:
Ah that's what @RKivioja had documented, maybe in the meantime the've released a new version